lib/wanda/messaging/mapper.ex

```diff
@@ -15,4 +17,23 @@ defmodule Wanda.Messaging.Mapper do
       {:ok, cloud_event}
     end
   end
+
+  @spec to_json(struct()) :: binary() | {:error, any()}
+  def to_json(event) do
```

Comment:
I don't think we need to return `:ok` or `:error` here, because conceptually nobody shouldn't be passing unsupported events to this function.

Reply:
I am not sure I fully understand. 
What do you mean? Could you elaborate a bit more?

Reply:
The caller of this function should know if an event is supported or not, hence the tagged tuple is not needed here IMHO